crates/futures/src/lib.rs

```diff
@@ -101,12 +101,22 @@
 //! }
 //! ```
 
+#![feature(stdsimd)]
```

Comment:
@alexcrichton it's not correct to leave it here, but when I put it inside `atomics.rs`, the example fails to compile, with error mentioning this feature

Reply:
Oh no this is correct yeah since the atomic intrinsics aren't stable yet, but you'll want to put this behind a `cfg_attr` to only compile it in when the `atomics` feature is enabled. That way this can continue to compile on stable.